recipes/qcustomplot/all/conanfile.py

```diff
@@ -76,7 +78,8 @@ def _patch_sources(self):
     def _configure_cmake(self):
         cmake = CMake(self)
         cmake.definitions["QT_VERSION"] = self.deps_cpp_info["qt"].version
-        # TODO: add an option to enable QCUSTOMPLOT_USE_OPENGL
+        if self.options.opengl:
+            cmake.definitions["QCUSTOMPLOT_USE_OPENGL"] = 1
```

Comment:
What do you expect from this CMake definition without a modification in the CMakeLists? Indeed, there is no QCUSTOMPLOT_USE_OPENGL option currently.